docs/sphinx/building/user_cookbook.rst

```diff
@@ -121,6 +121,34 @@ Centos 7 and later
     # install libraries, tools, and PulseAudio modules
     $ sudo scons -Q --enable-pulseaudio-modules --build-3rdparty=libuv,openfec,pulseaudio,cpputest install
 
+openSUSE Leap and later
+------------
```

Comment:
The line (`------------`) should be same length as the header:

```
openSUSE Leap and later
-----------------------
```

(this is the requirement of sphinx)